src/frontend/apps/standalone_site/src/features/Menu/store/menuStore.ts

```diff
@@ -1,14 +1,21 @@
 import create from 'zustand';
 
 interface UseMenu {
-  isMenuOpen: boolean;
+  isDesktopMenuOpen: boolean;
+  isMobileMenuOpen: boolean;
+  isMenuOpen: (breakpoint: string) => boolean;
```

Comment:
I'm not sure we really need 2 states

Reply:
I did use 1 state at the beginning, but in mobile version it is not very nice to have the menu open by default, so I updated the state when I get the Grommet responsive value, but we see a bit the menu and the burger icon moving when we load the page, even with `useLayoutEffect`, with this solution we don't update state so the display stay fixed.